### PR TITLE
Resolve spotlight/732 - kramdown security alert

### PR DESCRIPTION
**Pull request by iamjolly**
This PR includes the following updates:
- update jekyll to 4.1.1
- use newer ruby (to support newer jekyll)
- use newer bundler
- `bundle install`, `bundle update` with this branch resolves kramdown version security alert
- add querystring var to endpoint for mimetype to resolve 502 error
- removes two stray `endfor`s from uswds.md that were throwing build errors locally

Resolves [spotlight/732](https://github.com/18F/site-scanning/issues/732).

### CODE REVIEW

File: _plugins/jekyll_get.rb [hunk 1]
```diff
@@ -14,8 +14,8 @@ def generate(site)
       end
 
       # slurp down code.json data
-      url = site.config['scanner_url'] + 'search200/json/?200page=/code.json&present=Present'
-      data = JSON.load(open(url))
+      url = site.config['scanner_url'] + 'search200/json/?200page=/data.json&present=Present&mimetype=application/json'
+      data = JSON.load(URI.open(url))
```
Comment (alexbielen) [inline _plugins/jekyll_get.rb:18]: I'm not that familiar with Ruby, but can you tell me about this change? Was the URI library not in scope before?

**Review by alexbielen — COMMENTED** [review]
Just one quick question -- otherwise looks 👍

File: _plugins/jekyll_get.rb (hunk 1 above)
Comment (iamjolly) [reply _plugins/jekyll_get.rb:18]: I was getting this warning during build:
`...18f/site-scanning-dashboard/_plugins/jekyll_get.rb:18: warning: calling URI.open via Kernel#open is deprecated, call URI.open directly or use URI#open`

**Review by alexbielen — APPROVED** [review]
Looks good!

**Pull request merged**